demo/utils/CopyCodeButton.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <n-button
+    class="edit-button"
+    text
+    :size="size"
+    @click="handleClick"
+    :depth="depth"
+  >
+    <template #icon>
+      <n-icon>
+        <copy-icon />
+      </n-icon>
+    </template>
+  </n-button>
+</template>
+
+<script>
+import CopyIcon from '@vicons/fluent/Copy24Regular.js'
+import copy from 'copy-text-to-clipboard'
```

Comment:
Can you change it to a native implementation? I think we don't need to add a dep for this.

Reply:
ok